Makefile

```diff
@@ -4,7 +4,7 @@ update-version:
 	@perl -pi -e 's|<Version>[.\-\d\w]+</Version>|<Version>$(VERSION)</Version>|' src/Stripe.net/Stripe.net.csproj
 
 codegen-format:
-	dotnet format src/Stripe.net/Stripe.net.csproj
+	dotnet format src/Stripe.net/Stripe.net.csproj --severity warn
```

Comment:
why was this necessary?

Reply:
Makes format slightly faster.

Reply:
As it doesn't try to fix issues that won't result in an error.